cpp/include/cudf/detail/null_mask.hpp

```diff
@@ -58,6 +59,32 @@ void concatenate_masks(std::vector<column_view> const &views,
     bitmask_type * dest_mask,
     cudaStream_t stream);
 
+/**
+ * @brief Computes the required bytes necessary to represent the specified
+ * number of bits with a given padding boundary. 
+ * 
+ * @note Does not throw when passed an invalid padding_boundary
+ *
+ * @note The Arrow specification for the null bitmask requires a 64B padding
+ * boundary.
+ *
+ * @param number_of_bits The number of bits that need to be represented
+ * @param padding_boundary The value returned will be rounded up to a multiple
+ * of this value
+ * @return std::size_t The necessary number of bytes
+ **/
+CUDA_HOST_DEVICE_CALLABLE std::size_t bitmask_allocation_size_bytes_nocheck(size_type number_of_bits,
```

Comment:
The intent was to have a function that didn't throw so it could be called by a kernel.   The regular version of this function does a CUDF_EXPECTS() up front.  I could change the name back to  _nothrow instead of _nocheck.

Reply:
I see, I had thought [`div_rounding_up_safe`](https://github.com/rapidsai/cudf/blob/branch-0.13/cpp/include/cudf/detail/utilities/integer_utils.hpp#L114) could `throw`, but I was mistaken.